client/TracyProfiler.cpp

```diff
@@ -955,10 +975,13 @@ std::atomic<ThreadNameData*>& GetThreadNameData() { return GetProfilerData().thr
 TRACY_API LuaZoneState& GetLuaZoneState() { return GetProfilerThreadData().luaZoneState; }
 #  endif
 
+#  ifdef TRACY_MANUAL_LIFETIME
```

Comment:
`#ifndef`

Reply:
Ah, okay, sorry for violating the code style. Just for clarification, the reason I did it this way, is because in my experience this typically reads better, so you consistently have
```
#if ....
#else
#endif
```
and you always know that else block corresponds to not defined value. Otherwise, when readig you can misread `ifdef` and `ifndef` and get in trouble.

But of course, I will adhere to the coding guidelines and will fix that to `ifndef`